### DISCUSSION TITLE
reload pages from git storage

### DISCUSSION THREAD

**Discussion by vladisalv** [requarks/wiki#3372] [Help / Questions]
I have a new commit in my git repo with a new article. Git repo synchronizes well (I see green mark and I can manually check git log in wikijs docker container). If i do changes in old articles, everything is okay! But, wikijs doesn't see new files in git repo. I tried 'Force Sync' and 'Import Everything' options. But wikijs doesn't rebuild database with new articles. How can I fix it?

Ubuntu 18.04, docker
wiki.js 2.3.81

**Comment by mattiasJohnson** [comment]
You haven't found any solution have you? I'm in the same boat, commits to git repo in docker syncs with github but wont show up in my wiki
